src/Symfony/Component/Form/Extension/Validator/Type/UploadValidatorExtension.php

```diff
@@ -22,7 +22,14 @@
  */
 class UploadValidatorExtension extends AbstractTypeExtension
 {
+    /**
+     * @var TranslatorInterface
+     */
     private $translator;
```

Comment:
Yes it is private so no need php-doc. it is fixed. but i think it is easy for type hint when we write code. By the way is mention in http://symfony.com/doc/current/contributing/code/standards.html#naming-conventions :)